ios/pillarwallet/Info.plist

```diff
@@ -79,6 +79,14 @@
 		<string>SpaceGrotesk-Medium.ttf</string>
 		<string>PillarIcons.ttf</string>
 	</array>
+  <key>LSApplicationQueriesSchemes</key>
+  <array>
+      <string>mailto</string>
+      <string>airmail</string>
+      <string>ms-outlook</string>
+      <string>googlegmail</string>
+      <string>ymail</string>
+  </array>
```

Comment:
Are all of these array items supposed to be here?

Reply:
Here all items are not mandatory. it's the app through which we can send emails. I can change it according to your expectation.

Reply:
Removed it! mailto and whatsapp already exist in info.plist